apps/web/components/utils.tsx

```diff
@@ -269,3 +269,45 @@ export async function getAllCategories() {
     return [];
   }
 }
+
+export async function getAllMCQs() {
+  try {
+    const mcqs = await db.problem.findMany({
+      where: {
+        type: "MCQ",
+      },
+      include: {
+        mcqQuestions: true,
+      },
+    });
+    return mcqs;
+  } catch (e) {
+    return [];
+  }
+}
+
+export async function createMCQ(data: any) {
+  try {
+    const mcq = await db.mCQQuestion.create({
+      data,
+    });
+    return mcq;
+  } catch (e) {
+    console.log(e);
+    return null;
+  }
+}
+
+export async function deleteMCQ(id: string) {
+  console.log(id);
```

Comment:
same here, only admins should be allowed

Reply:
https://github.com/code100x/daily-code/pull/311/#discussion_r1573917619